css-basics/css-basics_combined-users/index.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <!-- add class="debug" to the html element to show helper texts -->
-<html lang="en" class="">
+<html lang="en" class="debug">
```

Comment:
Just out of curiosity, and to remember to ask you tomorrow about this class="debug"